content/en/docs/getting-started/reference-application-specification.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Specification for the reference application
+linkTitle: Reference App Spec
+description:
+  This document contains the specification for the reference application that is
+  used across the Getting Started guide
+toc_hide: true
```

Comment:
@svrnm - why hide this page from the sidenav? Is this page linked to anywhere else?